ygnmi/ygnmi_test.go

```diff
@@ -1395,6 +1395,27 @@ func TestLookupAll(t *testing.T) {
 		},
 		wantVals:             nil,
 		wantSubscriptionPath: leafPath,
+	}, {
+		desc: "non compliant value",
+		stub: func(s *testutil.Stubber) {
+			s.Notification(&gpb.Notification{
+				Timestamp: 100,
+				Update: []*gpb.Update{{
+					Path: testutil.GNMIPath(t, "model/a/single-key[key=10]/state/value"),
+					Val:  &gpb.TypedValue{Value: &gpb.TypedValue_IntVal{IntVal: 10}},
+				}, {
+					Path: testutil.GNMIPath(t, "model/a/single-key[key=11]/state/fake-val"),
```

Comment:
Is this a non-compliant value or just an irrelevant path given the subscription?

Reply:
if the server returns a path that doesn't match the subscription is considered a non-compliant error as well